desktop-widgets/mainwindow.ui

```diff
@@ -15,7 +15,16 @@
     <property name="spacing">
      <number>0</number>
     </property>
-    <property name="margin">
+    <property name="leftMargin">
+     <number>0</number>
+    </property>
+    <property name="topMargin">
+     <number>0</number>
+    </property>
+    <property name="rightMargin">
+     <number>0</number>
+    </property>
+    <property name="bottomMargin">
```

Comment:
I hate the noise that various versions of the designer create in the output

Reply:
Me too, I even re-did my QtCreator delete a second time, be be very sure I did nothing else then delete the part I wanted. By agree, I will filter the real change a leave everything else untouched.